website/views/options/social/challenges.jade

```diff
@@ -91,6 +91,19 @@ script(type='text/ng-template', id='partials/options.social.challenges.html')
             label
               input(type='radio', name='search-participation-radio', ng-click='search._isMember = undefined', checked='checked')
               =env.t('either')
+          h4=env.t('challengedOwnedFilterHeader')
+          .radio
+            label
+              input(type='radio', name='search-participation-radio', ng-click='search._isOwner = true')
+              =env.t('challengedOwnedFilter')
+          .radio
+            label
+              input(type='radio', name='search-participation-radio', ng-click='search._isOwner = false')
+              =env.t('challengedNotOwnedFilter')
+          .radio
+            label
+              input(type='radio', name='search-participation-radio', ng-click='search._isOwner = undefined')
```

Comment:
The ownership radio buttons should default to "Either" like Membership does:

![screen shot 2015-04-11 at 4 21 06 pm](https://cloud.githubusercontent.com/assets/2916945/7103319/d7d229ea-e066-11e4-8e8e-c51e9b38ed49.png)